### ISSUE TITLE
Crash of OCLP for MacBookPro4,1

### ISSUE THREAD

**Issue by cboukouv** [dortania/Opencore-Legacy-Patcher#84]
When I select as the target machine a MacBookPro4,1 which is supposed to be supported, the patcher crashes. This happens in all the versions of OCLP  that I have tested, and it can be easily reproduced on any Mac.

**Comment by khronokernel** [comment]
Fixed in main, was due to MacBookPro4,1 somehow getting mixed up with a Nvidia chipset. Here's a build with the .app if you don't have python 3 installed:

https://github.com/dortania/Opencore-Legacy-Patcher/suites/2214841459/artifacts/45823654

**Comment by cboukouv** [comment]
Thank you, it was indeed fixed and the EFI folder was properly generated. However the MacBook Pro is stuck on boot, and after trying to boot with the debug version of opencore, I get the error seen in the attached screen photo. Any ideas?

I have to say that the OCLP works fine on my MacBookPro 8,1 and my MacBookAir5,2

![IMG_0001](https://user-images.githubusercontent.com/63094758/110597067-c5437580-8188-11eb-88d4-499462a24516.jpeg)

**Comment by khronokernel** [comment]
Hmm makes sense, MacBookPro4,1 is a UGA era hardware but OpenCore expects to be GOP. Can you try this build and see if it resolves?:

* https://github.com/dortania/Opencore-Legacy-Patcher/suites/2225297312/artifacts/46090648

**Comment by cboukouv** [comment]
Apologies for not seeing your reply earlier. I just tested with the version that you have sent in the link and with the latest version 0.0.19 and with both I have the same problem. With the latest one I tried to boot in debug mode and I attach the opencore log, in case it helps. When I boot Catalina with dosdude1's patch I don't have this problem, although some text is displayed in the first stage of booting the MBP4,1.


[opencore-2021-03-27-165811.txt](https://github.com/dortania/OpenCore-Legacy-Patcher/files/6216058/opencore-2021-03-27-165811.txt)
